From: Simon McVittie <smcv@debian.org>
Date: Sun, 6 Dec 2009 23:44:36 +0000
Subject: [PATCH] Remove reference to picins from newlatex2e's base.tex

picins.sty is non-free and is not shipped by Debian. The only reference I
can find to it is in a "sidebar" command defined in base.tex, which is not
used anywhere else in base.tex or __init__.py, so hopefully removing it
won't hurt.
---
 docutils/writers/newlatex2e/base.tex |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/docutils/writers/newlatex2e/base.tex b/docutils/writers/newlatex2e/base.tex
index 4955e60..64a4d4f 100644
--- a/docutils/writers/newlatex2e/base.tex
+++ b/docutils/writers/newlatex2e/base.tex
@@ -81,8 +81,6 @@
   \usepackage{color}
   % Get \textnhtt macro (non-hyphenating type writer).
   \usepackage{hyphenat}
-  % For sidebars.
-  \usepackage{picins}
   % We use longtable to create tables.
   \usepackage{longtable}
   % Images.
@@ -801,7 +799,6 @@
 % Width.
 \Dprovidelength{\Dsidebarwidth}{0.45\linewidth}
 \providecommand{\DNsidebar}[1]{
-  \parpic[\Dsidebarposition]{%
     \begin{minipage}[t]{\Dsidebarwidth}%
       % Doing this with nested minipages is ugly, but I haven't found
       % another way to place vertical space before and after the fbox.
@@ -818,7 +815,6 @@
       }%
       \vspace{\Dsidebarvmargin}%
     \end{minipage}%
-  }%
 }
 
 
-- 
